### ISSUE TITLE
"Silverblue Guide" duplicates content in the rest of the docs

### ISSUE THREAD

**Issue by allanday** [fedora-silverblue/silverblue-docs#16]
The Silverblue documentation includes standard sections for things like installation and [upgrading](https://docs.fedoraproject.org/en-US/fedora-silverblue/upgrades/), but it also contains a [Silverblue Guide](https://docs.fedoraproject.org/en-US/fedora-silverblue/silverblue-guide/) which has its own sections covering the same topics (an example: [upgrading](https://docs.fedoraproject.org/en-US/fedora-silverblue/silverblue-guide/#_upgrades_and_rollbacks)).

We shouldn't nest a competing set of docs inside the docs.

I'll work on merging the "silverblue guide" into the rest of the docs, unless someone tells me otherwise.

**Comment by matthiasclasen** [comment]
that would be valuable

**Comment by allanday** [comment]
Done

**Comment by returntrip** [comment]
@allanday I am doing some small contributions to the silverblue-docs and was thinking about adding an how to section. I have noticed that some interesting "howtos" (i.e. automatic upgrades)  were part of the [`silverblue-guide.doc`](https://github.com/allanday/silverblue-docs/blob/19b87fb63b07924ea903b5d6d57ee7005bceac61/modules/ROOT/pages/silverblue-guide.adoc) which has now been removed. Is there a reason why those parts were removed and not reused/integrated in the rest of the documentation? I want to avoid creating documentation if it is not relevant or useful to the project, hence my question. Thanks.
